OpenOversight/tests/test_utils.py

```diff
@@ -305,3 +309,29 @@ def test_crop_image_calls_upload_image_to_s3_and_store_in_db_with_user_id(
             assert (
                 current_user.get_id() in upload_image_to_s3_and_store_in_db.call_args[0]
             )
+
+
+@pytest.mark.parametrize(
+    "units, has_officers_with_unit, has_officers_with_no_unit",
+    [
+        (["Not Sure"], False, True),
+        (["Donut Devourers"], True, False),
```

Comment:
💯 🍩 🚮

Reply:
The test data is fantastic
https://github.com/OrcaCollective/OpenOversight/blob/ca674ef880b91f6df1738287b3b13e76e94f270f/OpenOversight/tests/conftest.py#L302-L308